test/copy.tilelive.test.js

```diff
@@ -278,16 +278,17 @@ test('copy corrupt mbtiles', function(t) {
   });
 });
 
-test('error on invalid tile in mbtiles', function(t) {
+test('passes through invalid tile in mbtiles', function(t) {
```

Comment:
Just for clarity's sake, why is this mbtile file invalid? Would be nice to reference this in the future if we are more explicit with the test name / fixture name.